File: S/python/code.py
IPV4 = "IPv4"
IPV6 = "IPv6"
ERROR = "Error"

# return IPV4, IPV6 or ERROR constant
def check_ip_address(ip_to_check: str) -> str:
    # your code goes here
    return ERROR

ip_to_check = input()
print(check_ip_address(ip_to_check))

File: S/js/code.js
const IPV4 = "IPv4";
const IPV6 = "IPv6";
const ERROR = "Error";

function checkIpAddress(ip) {
    // your code goes here
    return ERROR;
}

const _readline = require('readline');


const _reader = _readline.createInterface({
    input: process.stdin
});

const _inputLines = [];
let _curLine = 0;

_reader.on('line', line => {
    _inputLines.push(line);
});

process.stdin.on('end', solve);

function solve() {
    const ip = readLine();
    console.log(checkIpAddress(ip));
}

function readInt() {
    const n = Number(_inputLines[_curLine]);
    _curLine++;
    return n;
}

function readLine() {
    const line = _inputLines[_curLine];
    _curLine++;
    return line;
}